index.js

```diff
@@ -1,4 +1,4 @@
 const NcbiSeqRetriever = require("./src/NcbiSeqRetriever");
-const retriever = new NcbiSeqRetriever();
+const retriever = new NcbiSeqRetriever(https://github.com/faryal92/ncbi-sequence-retriever/blob/a362a576e144568baf03fb990fd6fc01f6117191/package.json#L1);
```

Comment:
Thanks for your pull request. I am not sure why you want to make such a change. The constructor for NcbiSeqRetriever does not take any parameters. I think it does not make sense to add a parameter in this constructor. 
The constructor of NcbiSeqRetriever can be found in this link, which does not take any parameters.
https://github.com/FanPu1/ncbi-sequence-retriever/blob/master/src/NcbiSeqRetriever.js#L9-L14